### DISCUSSION TITLE
Installing app on Pixel Tablet via F-Droid

### DISCUSSION THREAD

**Discussion by S0ulf3re** [breezy-weather/breezy-weather#442] [General]
I have the Pixel tablet. And I'm trying to install Breezy weather from f-droid so I can easily update the app whenever a new version comes out. However most f-droid clients are restricting me from installing it because of the tablet not having a GPS. It seems to work just fine when I install an APK manually. So how can I get it from f-droid?

**Comment by papjul** [comment]
I have no idea why F-Droid would restrict that, the app is perfectly compatible with tablets and with devices without GPS. You can even have current location through an IP location service.

**Comment by S0ulf3re** [comment]
this is precisely what its complaining about. Hopefully the email
attachment shows up within GitHub

**Reply by papjul** [reply]
It doesn't

**Reply by S0ulf3re** [reply]
here

![Screenshot_20230902-131924](https://github.com/breezy-weather/breezy-weather/assets/50301900/ddd298a0-fd2f-4cf0-a109-82d724d67cfa)

**Reply by papjul** [reply]
Please report here: https://gitlab.com/IzzyOnDroid/repo/-/issues

**Reply by papjul** [reply]
Please link your created issue here, in case other people are interested. Thank you!

**Reply by S0ulf3re** [reply]
All right, done

https://gitlab.com/IzzyOnDroid/repo/-/issues/439

**Reply by S0ulf3re** [reply]
Alright, they said that they only provide the apk provided by the developer, so that's something needs to be done with the repo

https://gitlab.com/IzzyOnDroid/repo/-/issues/439#note_1544447589

**Reply by papjul** [reply]
Ok, it's curious Android let you install the app manually, so I thought it was a F-Droid-specific thing. I explicitly marked these features as non-required for v4.6.6 (next version), so if it still doesn't work when it gets released, that will be worth trying on the official F-Droid app.

**Comment by Mr-Bajs** [comment]
Installed it (version 4.6.7-beta) on a pixel tablet with GrapheneOS, from the IzzyOnDroid repo through F-Droid. It works fine now.
